netbox/dcim/choices.py

```diff
@@ -441,6 +443,9 @@ class PowerPortTypeChoices(ChoiceSet):
             (TYPE_USB_3_B, 'USB 3.0 Type B'),
             (TYPE_USB_3_MICROB, 'USB 3.0 Micro B'),
         )),
+        ('AC', (
```

Comment:
Not sure it makes sense to group this under "AC," since most of the preceding options are also AC power. Maybe just an "other" category at the end of the list?